resources/kiali/templates/kyma-additions/kcproxy-service.yaml

```diff
@@ -1,17 +1,14 @@
 apiVersion: v1
 kind: Service
 metadata:
-    name: {{ template "kiali.name" . }}-secured
-    labels:
-        app: {{ template "kiali.name" . }}-kcproxy
-        chart: {{ template "kiali.chart" . }}
-        heritage: {{ .Release.Service }}
-        release: {{ .Release.Name }}
+  name: {{ template "kiali.name" . }}-secured
+  labels:
+      {{- include "kiali.labels" . | nindent 4 }}
 spec:
-    ports:
-      - port: {{ .Values.kiali.spec.server.port }}
-        targetPort: {{ .Values.kcproxy.inPort }}
-        protocol: TCP
-        name: http
-    selector:
-        app: {{ template "kiali.name" . }}-kcproxy
+  ports:
+    - port: {{ .Values.kiali.spec.server.port }}
+      targetPort: {{ .Values.kcproxy.inPort }}
+      protocol: TCP
+      name: http
+  selector:
+      app: {{ template "kiali.name" . }}-kcproxy
```

Comment:
This label does not exist anymore

Reply:
Resolved on discussion: this label is still on the kiali pod (not the deployment), allowing the service to find it and so it has to stay as is.